src/cryptonote_core/tx_pool.cpp

```diff
@@ -847,8 +883,9 @@ namespace cryptonote
   {
     CRITICAL_REGION_LOCAL(m_transactions_lock);
     CRITICAL_REGION_LOCAL1(m_blockchain);
+    const relay_category category = include_sensitive_data ? relay_category::all : relay_category::broadcasted;
     tx_infos.reserve(m_blockchain.get_txpool_tx_count());
-    key_image_infos.reserve(m_blockchain.get_txpool_tx_count());
```

Comment:
This tx_infos reserve call likely wants the same include_sensitive_data/unrelated flag as key_image_infos.

Reply:
Yikes, missed the one above. I would've "cached" it on the stack had I noticed that ... updating.